tools/src/tester/types/story.types.ts

```diff
@@ -177,6 +164,23 @@ export interface Distributions {
   included?: DistributionsList;
   excluded?: DistributionsList;
 }
+/**
+ * Number of times to retry on error.
+ *
+ *
+ * This interface was referenced by `Story`'s JSON-Schema
+ * via the `definition` "Retry".
+ */
+export interface Retry {
+  /**
+   * Number of retries.
+   */
+  count: number;
+  /**
+   * Number of milliseconds to wait before retrying.
+   */
+  wait?: number;
+}
 /**
```

Comment:
[This file is not supposed to be edited by hand](https://github.com/opensearch-project/opensearch-api-specification/pull/772/files#diff-f957e97582dd9e6df4d998d5ba5491ba0b43103b1977e1eddc5671d1885ce9aeL14-L17)

Reply:
Yep, I reran the generator.